src/insights.ts

```diff
@@ -9,6 +9,8 @@ type BufferedMethodCall = [string, unknown];
 
 type SnippetAlgoliaInsights = BufferedMethodCall[];
 
+export type InsightsRegion = "de" | "us";
+
```

Comment:
Sorry, I think I missed this in the other PR, it makes more sense for this to be defined next to the `InsightsApiBeaconClient` as that is where the type is used. The rest of this code is merely passing the value down to that class.